### PR TITLE
Winter 20/21 newsletter

### PR DESCRIPTION
**Pull request by ElenaFdR**
This will be ready for review when the mailing list version is ready, to ensure that any changes are made to both versions in parallel.

### CODE REVIEW

**Review by Ainali — APPROVED** [review]
Yay! More news to the world!

**Comment by felixfaassen** [comment]
The reason why this is failing is not the fault of travis:

- ./_site/news/2021/03/04/newsletter-dec-2020-to-feb-2021.html

  *  internally linking to https//:schedule.mozillafestival.org/session/TDHQ3D-1, which does not exist (line 114)

     <a href="https//:schedule.mozillafestival.org/session/TDHQ3D-1">register here</a>


The URI is spelled incorrectly: **https//:**schedule.mozillafestival.org/session/TDHQ3D-1

it should be **https://**

File: _posts/2021-03-04-newsletter-dec-2020-to-feb-2021.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+We have 3 highlights:
+
+- We're delighted that the [Signalen codebase is now in incubation]({% post_url 2021-02-04-signalen-is-in-incubation %}) with the Foundation for Public Code.
+- We're proud to have launched a new product: a free [online course introducing the Standard for Public Code](https://citybycity.academy/course/standards-for-smart-cities), hosted by Open and Agile Smart Cities' CITYxCITY online learning platform.
+- We received a philanthropic gift from Josh Elkes - thanks for enabling our work, Josh!
+
+And coming up:
+
+- MozFest 2021: come discuss "How public code can transform the digital future of cities" with our President Ben Cerveny and Ger Baron, CTO of Amsterdam on March 11th at 19:15 [register here](https//:schedule.mozillafestival.org/session/TDHQ3D-1).
```
Comment (felixfaassen) [inline _posts/2021-03-04-newsletter-dec-2020-to-feb-2021.md:24]: URI misspelled - change to **https://*

**Review by felixfaassen — CHANGES_REQUESTED** [review]

File: _posts/2021-03-04-newsletter-dec-2020-to-feb-2021.md (hunk 1 above)
Comment (ElenaFdR) [reply _posts/2021-03-04-newsletter-dec-2020-to-feb-2021.md]: Amazing spot, thank you!

**Review by felixfaassen — APPROVED** [review]
GREAT!

**Pull request merged**